extensions/wikia/Wall/index/CommentsIndexEntry.class.php

```diff
@@ -0,0 +1,348 @@
+<?php
+
+/**
+ * Class CommentsIndexEntry represents a single row in comments_index table.
+ * Each property corresponds to a column in comments_index table
+ *
+ * @see CommentsIndexEntry::getDatabaseRepresentation()
+ */
+class CommentsIndexEntry {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Class "CommentsIndexEntry" has 32 methods, which is greater than 20 authorized. Split it into smaller classes. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1448)